### ISSUE TITLE
Теряются подписки в mqtt после переподключения WiFi

### ISSUE THREAD

**Issue by georotor** [slsys/Gateway#172]
Прошивка 2023.02.15d7.

В `init.lua` прописаны подписки на mqtt вида: `mqtt.sub("cc1101/received", "cc1101.received")`
Заметил что периодически SLS перестаёт обновлять привязанные к топикам объекты, но при этом исправно публикует свои данные.

Поиски показали, что это происходит после падения сети WiFi, при восстановление соединения с брокером подписки теряются. Пробовал сделать отдельный скрипт в котором указаны необходимые `mqtt.sub...` и запустить вручную, но это не помогает, спасает только перезагрузка шлюза.

Явление нечастое, но очень неприятное.

**Comment by Avenitos** [comment]
Да, при переподключении к MQTT сейчас нет восстановления скриптовых подписок, добавил в список TODO
